builtin/providers/google/provider.go

```diff
@@ -21,7 +21,7 @@ func Provider() terraform.ResourceProvider {
 				Optional:     true,
 				DefaultFunc:  schema.EnvDefaultFunc("GOOGLE_ACCOUNT_FILE", nil),
 				ValidateFunc: validateAccountFile,
```

Comment:
Do you still need the ValidateFunc if the field itself is Removed? If not then you can remove the function entirely.